NetKAN/KSP-Slovakia-flags.netkan

```diff
@@ -0,0 +1,22 @@
+{
+    "spec_version" : "v1.2",
+    "identifier"   : "KSP-Slovakia-flags",
+    "$kref"        : "#/ckan/github/miki-g/KSP-Slovakia-Flags",
+    "ksp_version"    : "any",
+    "comment"        : "flagmod",
+    "name"         : "KSP Slovakia flags",
+    "author"       : [ "Miki" ],
+    "abstract"     : "Slovakia Flags",
+    "license"      : "WTFPL",
+
+    "install": [
+          {
+           "file" : "KSP Slovakia flags",
```

Comment:
"file" : "KSP Slovakia Flags",

should work. Case matters for Linux file systems.